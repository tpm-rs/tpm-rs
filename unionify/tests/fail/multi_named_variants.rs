use tpm2_rs_unionify::UnionSize;

#[derive(UnionSize)]
pub enum Foo {
    A { x: u8, y: u8 },
    B(u64),
}

fn main() {}
