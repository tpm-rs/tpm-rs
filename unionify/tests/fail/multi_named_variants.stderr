error: UnionSize is not derivable for enum variants containing more than one field
 --> tests/fail/multi_named_variants.rs:5:7
  |
5 |     A { x: u8, y: u8 },
  |       ^^^^^^^^^^^^^^^^
